libs/transactions/src/lib/services/transactions-route-title-resolver.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import {
+  ActivatedRouteSnapshot,
+  Resolve,
+  RouterStateSnapshot,
+} from '@angular/router';
+import { of } from 'rxjs';
+import { TransactionsJourneyConfiguration } from './transactions-journey-config.service';
+
+@Injectable()
+export class TransactionsRouteTitleResolverService implements Resolve<unknown> {
```

Comment:
- [ ] Please add UT

Reply:
I am not sure if this is really testable. Because this service is used by the internals of angular routing module

Reply:
At the end I did some simple unit test for this. Please take another look